README.md

```diff
@@ -42,13 +44,13 @@ yarn start-wallet-connect
 
```

Comment:
and also in line 34 add the commands are now:
```bash
yarn start:compound
yarn start:drain-safe
yarn start:tx-builder
yarn start:walletconnect
```
And also add the new ramp safe app script:
```
yarn start:ramp
```

Reply:
This is already in place, but we are missing the merge from development. Will do